vllm/utils.py

```diff
@@ -27,7 +27,6 @@
 from typing_extensions import ParamSpec
 
 import vllm.envs as envs
-from vllm import _custom_ops as ops
```

Comment:
Why this was removed? This might break line: https://github.com/HabanaAI/vllm-fork/pull/337/files#diff-dab7693bd00a09e22d39aee684a7e419aa358a47c4bd20df33d44f5adf60d304R247

Reply:
i removed this because it was complaining about circular imports.

Reply:
closing as obsolete, given #293 is merged